app/src/main/java/com/github/se/icebreakrr/ui/profile/ProfilePictureSelector.kt

```diff
@@ -43,9 +48,11 @@ private val ICON_PADDING = 5.dp
 fun ProfilePictureSelector(
     url: String?,
     localBitmap: Bitmap?,
+    pictureChangeState: ProfilesViewModel.ProfilePictureState,
     size: Dp,
     onSelectionSuccess: (Uri) -> Unit,
-    onSelectionFailure: () -> Unit
+    onSelectionFailure: () -> Unit,
+    onDeletion: () -> Unit,
```

Comment:
Lack of documentation for those new parameters

Reply:
That one was missed even after my documentation update, thanks!